src/dockerfile.mli

```diff
@@ -138,7 +156,7 @@ val env : (string * string) list -> t
   instructions. This is functionally equivalent to prefixing a shell
   command with [<key>=<value>]. *)
 
-val add : ?chown:string -> ?from:string -> src:string list -> dst:string -> unit -> t
+val add : ?link:bool -> ?chown:string -> ?from:string -> src:string list -> dst:string -> unit -> t
 (** [add ?from ~src ~dst ()] copies new files, directories or remote file URLs
   from [src] and adds them to the filesystem of the container at the
   [dst] path.
```

Comment:
Maybe here add the  `?link` with a small explanation about what it does?

Reply:
Agree, please document what `?link:bool` does.